build merge doesn't fail gracefully when there is an issue.

For example [here](https://ci.centos.org/job/devtools-fabric8-build-fabric8-push-build-master/10/console) :

```
buildah build -t quay.io/openshiftio/fabric8-services-fabric8-build -f ./.make/Dockerfile .
No help topic for 'build'
make: *** [image] Error 3
++ echo 9c0cd8f941afe7e7ddbd299e3772d22f7793c209
++ cut -c1-7
+ TAG=9c0cd8f
```

Should fail and not go on

--- a/.cico/setup.sh
+++ b/.cico/setup.sh
@@ -57,8 +57,7 @@ function tag_push() {
     buildah push --creds "${QUAY_USERNAME}:${QUAY_PASSWORD}" ${image}:${tag} ${image}:${tag}
 }
 
-
-function _deploy() {
+function deploy() {
   # Login first
   cd ${REPO_PATH}
 
@@ -75,17 +74,6 @@ function _deploy() {
   fi
 
   echo 'CICO: Image pushed, ready to update deployed app'
-}
-
-
-function deploy() {
-    set +e
-    _deploy || fail=true
-    set -e
-
-    if [[ -n ${fail} ]];then
-		echo "We need to find a way to notify that something has failed"
-    fi
 }
 
 function check_up() {

